### ISSUE TITLE
Support for [adding custom logs on log analytics workspace]

### ISSUE THREAD

**Issue by Terraformazure** [terraform-providers/terraform-provider-azurerm#8018]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* azurerm_log_analytics_workspace

### References

https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-sources-custom-logs

* #0000

**Comment by magodo** [comment]
Hi @Terraformazure Thank you for submitting this :+1:

There is another issue #3182 requring kinds of log analytics workspace data sources. The process is currently blocked by Azure/azure-rest-api-specs#9072. Once that issue has been addressed, we can move on to implementing the remaining data sources.

In order not to duplicate issues, hope you don't mind I close this issue in favor of #3182.
